apps/chatbot/lint-staged.config.cjs

```diff
@@ -0,0 +1,9 @@
+const mainConfig = require("../../lint-staged.config.js");
+
+const buildEslintCommand = (filenames) =>
+  `pnpm run lint --fix ${filenames.join(" ")}`;
```

Comment:
`pnpm run lint` is an alias to `eslint .`, so it will always run against all files, even if you're passing extra file names here. This should probably just interface with eslint directly.